back/app/models/user.rb

```diff
@@ -133,11 +135,15 @@ def from_token_request(request)
   before_validation :generate_slug
   before_validation :sanitize_bio_multiloc, if: :bio_multiloc
   before_validation :assign_email_or_phone, if: :email_changed?
+  with_options if: -> { user_confirmation_enabled? } do
+    with_options if: :email_changed?, on: :create do
+      before_validation :reset_confirmation_required
+    end
 
-  has_many :notifications, foreign_key: :recipient_id, dependent: :destroy
-  has_many :unread_notifications, -> { where read_at: nil }, class_name: 'Notification', foreign_key: :recipient_id
+    before_validation :confirm, if: ->(user) { user.invite_status_change&.last == 'accepted' }
```

Comment:
I think this will always confirm the email of a user who accepted their invitation, even if they changed their email afterwards? Not 100% sure, but I think it was also possible to change the email while an invitee accepts their invitation.

Reply:
Yeah I just moved it from where it was before, so there will be no difference. Will make a note to test for this though.